lobsterpy/structuregraph/__init__.py

```diff
@@ -0,0 +1,154 @@
+# Copyright (c) lobsterpy development team
+# Distributed under the terms of a BSD 3-Clause "New" or "Revised" License
+
+"""
+This package provides the modules for generating graph objects using lobsterpy data
+"""
+
+import os
+from pymatgen.core.structure import Structure
+from pymatgen.io.lobster.lobsterenv import LobsterNeighbors
+from pymatgen.io.lobster.outputs import Charge
+from lobsterpy.cohp.analyze import Analysis
+
+
+class GraphObject:
+    """
+    GraphObject class for bonding data from Lobster
+
+    Attributes:
+        sg: return structure_graph object
+    """
+
+    def __init__(
+        self,
+        path_to_calcdir: str,
```

Comment:
I think there is a chance that people have the files in different folders. I think we should support it.

Reply:
Yes, I thought it would be bit faster to use, and number of argument lists is bit large to type it out. But what you said also makes sense. So will revert it back to initial style.